stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -1291,6 +1291,10 @@ function(_add_swift_target_library_single target name)
         endif()
       endif()
     endif()
+
+    # Silence warnings about global initializers. We already have clang
+    # emitting warnings about global initializers when it compiles the code.
+    list(APPEND swiftlib_link_flags_all "-Wl,-no_warn_inits")
```

Comment:
This needs to be guarded by a Darwin check as this is the common path (unless indentation of this ginormous function is getting me confused).

Nit: I think that we should prefer to use `-Xlinker -no_warn_inits` over `-Wl,-no_warn_inits` as the former is more modern and makes it easier to find the arguments.

Reply:
It's inside a `if(${SWIFTLIB_SINGLE_SDK} IN_LIST SWIFT_APPLE_PLATFORMS)` block, is that sufficient?

I'm OK with switching to `-Xlinker`, I just cargo-culted the `-Wl,` version from another spot.

Reply:
Ah, that is difficult to line up, the CMake here is so difficult to follow :-(